WordPressKit/WordPressComRestApi.swift

```diff
@@ -357,8 +357,8 @@ open class WordPressComRestApi: NSObject {
                         progress.completedUnitCount = progress.totalUnitCount
                         success(responseObject as AnyObject, response.response)
                     case .failure(let error):
-                        let nserror = self.processError(response: response, originalError: error)
-                        failure(nserror, response.response)
+                        let nserror = self.processError(response: response, originalError: error).flatMap { $0 as NSError }
+                        failure(nserror ?? (error as NSError), response.response)
```

Comment:
The original naming of the value made me feel confused when reading `(error as NSError)`.

What do you think of `processedError`

```suggestion
                        let processedError = self.processError(response: response, originalError: error).flatMap { $0 as NSError }
                        failure(processedError ?? (error as NSError), response.response)
```